assets/styles/homepage.scss

```diff
@@ -64,6 +64,10 @@ p {
   padding-top: 6rem;
   text-align: center;
 
+  @media screen and (min-aspect-ratio: 10/7) {
+    background-image: url('../images/houses-wide.svg');
+  }
+
   @media (max-height: 400px) {
     background-image: none;
```

Comment:
after this line, a `padding-top` smaller than `6em` would have the text visually center a bit better.